main.js

```diff
@@ -9,7 +9,7 @@ function run() {
 
         shell.env["DEVELOPER_DIR"] = xcode_dir;
         shell.exec("sudo systemsetup -settimezone 'Europe/Stockholm'");
-
+        shell.exec("bundle install");
```

Comment:
Because we are adding 2 new gems for the metadata release process: 
https://github.com/voiapp/hunter_ios/pull/476/files#diff-d09ea66f8227784ff4393d88a19836f321c915ae10031d16c93d67e6283ab55fR1

Reply:
If we don't run `bundle install` we won't be able to download those ruby packages